src/en/config-gce.md

```diff
@@ -1,255 +1,201 @@
-this elsewhere, so don't make it too unwieldy)
+The *project id* (used later) will be generated automatically. Use the 'Edit'
+button to change it.
 
-![image showing project details](media/config-gce03.png)
 
+## Google Compute Engine API
 
-## 3. Enable the Google Compute Engine API
+The Google Compute Engine API needs to be enabled for your new project in order
+for Juju to communicate with it. This is done automatically if a "billing
+method" has been set up. By following the below steps you will discover whether
+you need to set up billing or not. Check to see if a trial account is available.
 
-Juju requires the Google Compute Engine API to be active for the project. This is _usually_ activated by default, but it is worth checking at this point. From the Main console Window, select `APIs & auth` and then `APIs` to see the list of available APIs.
+At the top-left of the web UI there is an icon representing 'Product &
+services'. It is denoted by this icon:
 
-![image showing APIs](media/config-gce-extra01.png)
+![Product & services icon](./media/config-gce-product_services_icon.png)
 
-Click on Compute Engine API, the first in the list.
-On the new screen which opens, if there is a button titled `Enable` (as shown
-below), then you should click it. If the API is already enabled, the button will
-show `Disable Api` instead, as well as details of your usage. Don't disable it!
+Click through and select 'API Manager'. By default you will be on the 'Overview'
+screen, it will show:
 
-![image showing Enable option](media/config-gce-extra02.png)
+![API Manager screen](./media/config-gce-api_manager.png)
 
+Select 'Compute Engine API' to get:
 
-## 4. Generate OAuth credentials for the project
+![Compute Engine API](./media/config-gce-api_manager_compute_enabled.png)
 
-You will be returned to the main console
-screen. In order to let Juju have access to the environment, you will need to
-generate and retrieve some credentials. In the navigation on the left, you
-should select `APIs & auth`, and then select `Credentials` from the panel which
-will appear.
+The API is already enabled if the button displays 'Disable API'. If the button
+shows 'Enable API', clicking it may prompt you to set up a billing method (if
+not already done).
 
-![image showing main screen](media/config-gce04.png)
 
-On the credentials page itself, select 'Create new Client ID'
+## GCE project credentials file
 
-![image showing API page](media/config-gce05.png)
+A collection of credentials-related material will be required. This is obtained
+by downloading a file from the UI.
 
-A pop-up will appear asking which type of credential to generate. You *must* 
-select the `Service account` option - Juju needs these credentials to be able to
-create new instances.
+Return to the 'API Manager' and choose the 'Credentials' screen. By default you
+will be within the 'Credentials' tab. Click the 'New credentials' button and
+choose 'Service account key' among the 3 options available:
```

Comment:
is this quirky Canadian grammar? In blighty, we always choose from, or maybe at a push, chose from among